content/en/docs/concepts/security/index.md

```diff
@@ -190,6 +190,15 @@ follows:
 1. After authorization, the server side Envoy forwards the traffic to the
    server service through local TCP connections.
 
+Istio configures TLSv1_2 as the minimum TLS version for both client and server with 
+the following cipher suites:
```

Comment:
Fix lint error (trailing space on line 193). Also you need a blank line after 194, or the following list items will not display properly in [the preview](https://deploy-preview-8420--preliminary-istio.netlify.app/latest/docs/concepts/security/#mutual-tls-authentication).
```suggestion
Istio configures TLSv1_2 as the minimum TLS version for both client and server with
the following cipher suites:

```